mqtt-protocol/src/jsMain/kotlin/it/unibo/pulvreakt/mqtt/MqttClient.kt

```diff
@@ -0,0 +1,69 @@
+@file:Suppress("REDUNDANT_NULLABLE")
```

Comment:
If the compiler raises a redundant nullable, you can remove the nullability.
Maybe it's related to the `dynamic` which implicitly assumes the nullability?
Could you check this aspect?